util/src/dbutil.cpp

```diff
@@ -224,5 +281,32 @@ std::shared_ptr<odb::database> createDatabase(const std::string& connStr_)
   return db;
 }
 
+void createTables(
+  std::shared_ptr<odb::database> db_,
+  const std::string& sqlDir_)
+{
+  runSqlFiles(db_, sqlDir_,
+    [](const std::string& s){
```

Comment:
Use underscore at the end of function parameters:
```cpp
[](const std::string& s_){
```

Same at `createIndexes()` function.